src/pug/pages/products-list.pug

```diff
@@ -0,0 +1,27 @@
+extends ../base/base-products
+
+block title 
+  title Products
+
+block content
+  .products-list.gx-5
+    each item in products
+      if item.onPage1
+        .products-list__item.row
+          .col-md-4
+            img.img-fluid.mx-auto.d-block(src=item.src, alt="product image")
+            hr.d-md-none
+          .col-md-8.d-flex.flex-column
+            p.bold.product-card__name= item.name
+              span.bold.product-card__name &nbsp;thương hiệu&nbsp;
```

Comment:
Sao phải thêm 2 dấu space như này, padding thì sao

Reply:
Vì cả p và span ở đây là cùng 1 line nên khi dùng "&nbsp;" thì kích thước space sẽ đồng bộ với font-size 1em, và nó cũng sẽ cố định nữa (vì space trong 1 line of text ko bao h đổi). Đỡ dc thêm vài dòng css.